packages/realtime_client/lib/src/realtime_channel.dart

```diff
@@ -180,8 +180,16 @@ class RealtimeChannel {
     }
   }
 
-  void subscribe(
-      [void Function(String, [Object?])? callback, Duration? timeout]) {
+  /// Subscribe registers your client with the server
```

Comment:
I'm not sure what this sentence means. Do you mean with "Subscribe" the method name?  Then I would make it lower case and wrap it in `[]` or remove it completely.

Reply:
That's better. Sometimes I see `realtime` and sometimes `real-time`. Is there any reason or are both ways fine?

Reply:
Yeah, "Realtime" is a name of a service that Supabase offers, and "real-time" is a term to describe multiple events happening simultaneously. "realtime" is the grammatically incorrect version of "real-time". I'm sure within Supabase products, there are tons of those typos though 😂